c2corg_ui/templates/article/preview.html

```diff
@@ -0,0 +1,36 @@
+<%namespace file="../helpers/common.html" import="show_title, show_fulldate"/>
+<%namespace file="helpers/detailed_article_attributes.html" import="get_article_general"/>
+<%namespace file="../helpers/view.html" import="get_document_locale_text, show_attr, get_document_description, get_route_activities"/>
+<%namespace file="../helpers/preview.html" import="show_preview_warning"/>
+
+<%
+article = document
+locale = document['locales'][0]
+%>
+
+${show_preview_warning()}
+
+<header class="view-details-title">
+  <h1 class="routes">
+    <div class="icons-header routes">${get_route_activities(article, 'top')}</div>
+    <span class="title">${locale['title']}</span><br>
+    <label class="badge article">
+      <span class="glyphicon glyphicon-arrow-right"></span>&nbsp;<span translate>article</span>
+    </label>
+  </h1>
+</header>
+
+<section class="view-details-section" app-view-details>
```

Comment:
Could you remove ``app-view-details`` here, as in this pending PR: https://github.com/c2corg/v6_ui/pull/740/files ?